CRM/Utils/File.php

```diff
@@ -973,7 +973,13 @@ public static function getFileURL($path, $fileType, $url = NULL) {
   public static function getImageURL($imageURL) {
     // retrieve image name from $imageURL
     $imageURL = CRM_Utils_String::unstupifyUrl($imageURL);
-    parse_str(parse_url($imageURL, PHP_URL_QUERY), $query);
+
```

Comment:
How about just `parse_str(parse_url($imageURL, PHP_URL_QUERY) ?? '', $query);` ?

And then also don't have to deal with the style error that jenkins doesn't like :)

Reply:
Ah crap, missing space before the `{` innit. Okay, I'll replace with your suggestion 😎

Reply:
Done - cheers for reviewing @seamuslee001